### DISCUSSION TITLE
e107sk.com new theme...

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5166] [Show and tell]
With the new year, a new theme for e107.sk

![image](https://github.com/e107inc/e107/assets/5429548/aabd7456-348f-435c-97a8-6208cfaac529)

**Comment by Alex-e107nl** [comment]
@Jimmi08 hi i do not get the site, only preloader... 
![preloader](https://github.com/e107inc/e107/assets/11962513/aefcd04c-aee7-47cc-9ee8-54394ca34f6f)

Would like to see it now :-)

**Reply by Jimmi08** [reply]
there was no change.  It works normally https://www.e107sk.com/  check console please

**Comment by Alex-e107nl** [comment]
Solved! Nice!

**Comment by LaocheXe** [comment]
@Jimmi08 love the new theme, love the switch between light and dark. Keep up the epic work!
